app/services/course_search_service.rb

```diff
@@ -11,9 +11,7 @@ def call(**args)
   end
 
   def call
-    scope = Course
-      .findable
-      .with_recruitment_cycle(recruitment_cycle_year)
+    scope = course_scope.findable
```

Comment:
I feel like `findable` should actually be passed in by the caller, since different controllers could potentially not care about `findable`, and since we have the mechanism available let's use it?

Reply:
I wondered about that. Could we move it when we need to add another caller that doesn't want just findable courses rather than now when the only use we have is findable only?

Reply:
fair call on YAGNI

Reply:
So long as we don't forget about it at a later time and introduce a subtle bug. :)

Reply:
true, but it will be along the line with introduce new feature and introduce a bug that did not expect the course search to only return `findable`